### PR TITLE
Updates the list of files to hide in Mage apps

### PR DESCRIPTION
**Pull request by infomiho**
Due to new project structure, we need to updated which files are hidden by default when previewing the apps in the Mage app. Mostly, those are configuration apps that are not interesting to look at.

### CODE REVIEW

File: mage/src/client/pages/ResultPage.jsx [hunk 1]
```diff
@@ -490,7 +491,9 @@ export function OnSuccessModal({ isOpen, setIsOpen, appGenerationResult }) {
                 <td className="p-2 text-gray-600"> Cost to generate your app: </td>
                 <td className="p-2 text-gray-600">
                   {" "}
-                  <FormattedText>{`~$${calcCostForGpt_3_5_Turbo_0125(Number(numTokensSpent))}`}</FormattedText>{" "}
+                  <FormattedText>{`~$${calcCostForGpt_3_5_Turbo_0125(
```
Comment (infomiho) [inline mage/src/client/pages/ResultPage.jsx:494]: Autoformat

File: mage/src/client/pages/ResultPage.jsx [hunk 2]
```diff
@@ -456,13 +455,15 @@ export function OnSuccessModal({ isOpen, setIsOpen, appGenerationResult }) {
...
 
-  function calcCostForGpt_3_5_Turbo_0125 (numTokensSpent) {
+  function calcCostForGpt_3_5_Turbo_0125(numTokensSpent) {
     const estimatedInputTokenShare = 0.8;
     const estimatedOutputTokenShare = 1 - estimatedInputTokenShare;
-    const costInUsdForMillionInputTokens = 0.5;  // This is price for GPT 3.5 Turbo 0125.
-    const costInUsdForMillionOutputTokens = 1.5;  // This is price for GPT 3.5 Turbo 0125.
-    const costInUsdForMillionTokens = costInUsdForMillionInputTokens * estimatedInputTokenShare + costInUsdForMillionOutputTokens * estimatedOutputTokenShare;
-    return (numTokensSpent / 1e6 * costInUsdForMillionTokens).toFixed(2);
+    const costInUsdForMillionInputTokens = 0.5; // This is price for GPT 3.5 Turbo 0125.
```
Comment (infomiho) [inline mage/src/client/pages/ResultPage.jsx:461]: Autoformat

File: mage/src/client/pages/ResultPage.jsx [hunk 3]
```diff
@@ -139,12 +139,11 @@ export const ResultPage = () => {
...
-            path !== "src/client/vite-env.d.ts" &&
-            path !== "src/client/tsconfig.json" &&
-            path !== "src/server/tsconfig.json" &&
-            path !== "src/shared/tsconfig.json" &&
+            path !== "src/vite-env.d.ts" &&
+            path !== "tsconfig.json" &&
             path !== ".gitignore" &&
-            path !== "src/.waspignore" &&
+            path !== ".waspignore" &&
+            path !== "public/.gitkeep" &&
```
Comment (sodic) [inline mage/src/client/pages/ResultPage.jsx:146]: What about package-lock.json? We want to show that?

Comment (infomiho) [reply mage/src/client/pages/ResultPage.jsx:146]: We are not running the apps, so no package-lock.json is generated for now.

**Review by sodic — APPROVED** [review]
Went through the docs and restructuring changes to double check the file list - seems to me all's well.

**Pull request merged**